src/components/cookie-notice/cookie-notice.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div class="cookie-notice">
```

Comment:
Bit of an odd request but could we rename this to something else?
Since it contains `cookie` my adblocker removes the whole element so there's no video at all.